cmd/tengo/main.go

```diff
@@ -55,10 +55,34 @@ func main() {
 	inputData, err := ioutil.ReadFile(inputFile)
 	if err != nil {
 		_, _ = fmt.Fprintf(os.Stderr,
-			"Error reading input file: %s", err.Error())
+			"Error reading input file: %s\n", err.Error())
 		os.Exit(1)
 	}
 
+	if compileOutput != "" {
```

Comment:
do we have to resolve `compileOutput` path?

Reply:
os.Chdir changes working dir below this line; otherwise relative output like this `tengo -o ./a/dir/a/file input.tengo` points to wrong location. Getting absolute path before `os.Chdir` fixes this problem. I am open to suggestions.

Reply:
This will be removed after removing `os.Chdir` :+1: